mods/d2k/maps/atreides-04/atreides04-AI.lua

```diff
@@ -0,0 +1,163 @@
+IdlingUnits = { }
+
+AttackGroupSize =
+{
+	easy = 6,
+	normal = 8,
+	hard = 10
+}
+
+AttackDelays =
+{
+	easy = { DateTime.Seconds(4), DateTime.Seconds(7) },
+	normal = { DateTime.Seconds(2), DateTime.Seconds(5) },
+	hard = { DateTime.Seconds(1), DateTime.Seconds(3) }
+}
+
+HarkonnenInfantryTypes = { "light_inf", "light_inf", "trooper", "trooper", "trooper" }
+HarkonnenVehicleType = { "combat_tank_h" }
+
+AttackOnGoing = false
```

Comment:
Those are different things. `AttackOnGoing` is used by `DefendActor` while `SendAttack` uses `Attacking`.
Should I rename `AttackOnGoing` to `Defending`?